src/Agent.Worker/Build/FileContainerServer.cs

```diff
@@ -345,7 +345,13 @@ private async Task<UploadResult> BlobUploadAsync(IAsyncCommandContext context, I
                 var verbose = String.Equals(context.GetVariableValueOrDefault("system.debug"), "true", StringComparison.InvariantCultureIgnoreCase);
                 int maxParallelism = context.GetHostContext().GetService<IConfigurationStore>().GetSettings().MaxDedupParallelism;
                 (dedupClient, clientTelemetry) = await DedupManifestArtifactClientFactory.Instance
-                    .CreateDedupClientAsync(verbose, (str) => context.Output(str), this._connection, maxParallelism, token);
+                    .CreateDedupClientAsync(
+                    verbose,
+                    (str) => context.Output(str),
+                    this._connection,
+                    maxParallelism,
+                    Microsoft.VisualStudio.Services.BlobStore.WebApi.Contracts.Client.BuildArtifact,
+                    token);
```

Comment:
is this the build artifact using blobstore as backend?

Reply:
yes